cmake/kokkos_tribits.cmake

```diff
@@ -244,6 +244,15 @@ FUNCTION(KOKKOS_SET_LIBRARY_PROPERTIES LIBRARY_NAME)
     ${LIBRARY_NAME} PUBLIC ${KOKKOS_LINK_LIBRARIES}
   )
 
+  IF (KOKKOS_ENABLE_OPENMP)
+
+    TARGET_LINK_LIBRARIES(
+      ${LIBRARY_NAME} PUBLIC OpenMP::OpenMP_CXX
+    )
+
+  ENDIF()
```

Comment:
I suspect we would need to update the build with Trilinos as well.  @jjwilke what would be the right way to do this?

Reply:
Wait for Trilinos to move to a target-properties based CMake... This is fine for building Kokkos and projects within Trilinos.  The challenge here is the exporting KokkosConfig.cmake